src/core/dyad_core.c

```diff
@@ -0,0 +1,622 @@
+#include "dyad_core.h"
+#include "dyad_err.h"
+#include "utils.h"
+#include "murmur3.h"
```

Comment:
I guess these are not installed headers. So, it is ok to be without the subfoler location of headers. If it was cmake, it would have been better to match the header files with the actual location of headers. e.g., "common/murmur3.h"